packages/tempest/styleMock.ts

```diff
@@ -0,0 +1 @@
+export default {}
```

Comment:
Let's add a comment explaining why this needs to be here- it's a bit confusing at first glance if you don't know about the jest configuration problems

Reply:
will do